masonry/src/event.rs

```diff
@@ -50,6 +50,84 @@ pub enum PointerButton {
     Other,
 }
 
+/// A set of [`PointerButton`]s.
+#[derive(PartialEq, Eq, Clone, Copy, Default)]
+pub struct PointerButtons(u8);
```

Comment:
Can we have the same utils as from bitflags (such as `|`, etc.)

I know we don't want to have different public names for these versions of the values (which is right), so we can't really use bitflags directly.

In theory, I guess we could make `PointerButton | PointerButton` yield `PointerButtons`, but I think that's too cute.

Reply:
We barely use `PointerButtons` right now (only on the `PointerState`).

If I did this, I'd want to move it to a separate file as it would start to grow too large to just have it in `event.rs` ... logically, we could do `masonry/src/event/pointer.rs` and then `pub use` the stuff from it into `masonry::event`. It would be easiest then if we left `src/event.rs` rather than renaming it to `src/event/mod.rs` ...

Reply:
This also starts to get closer to where I'd rather have a new crate called `input-types` (name not taken yet) or something and go all out there since this isn't specific to Masonry (and I have another version of code like this in my own codebases...)